config/routes.rb

```diff
@@ -142,15 +142,18 @@
       get   "/settings" => "communities#settings",        as: :settings
       patch "/settings" => "communities#update_settings", as: :update_settings
 
+      # Guide
       get "getting_started_guide(/*all)" => "getting_started_guide#index", as: :getting_started_guide
 
+      # Details and look 'n feel
+      get   "/look_and_feel/edit" => "communities#edit_look_and_feel"
+      patch "/look_and_feel"      => "communities#update_look_and_feel"
+      get   "/details/edit"       => "community_customizations#edit_details"
+      patch "/details"            => "community_customizations#update_details"
```

Comment:
I think it would be better style to explicitly name the routes with :as to make it clear what the path/url helpers are called.